controller/controller.go

```diff
@@ -117,6 +117,11 @@ func (ctrl *baseController) init(ctx context.Context) (*driver.Drivers, error) {
 			return nil, err
 		}
 
+		err = d.ValidateTask(ctx)
```

Comment:
Looks like a good location to validate! Right after all of the files are generated 👍